sys/include/xtimer/implementation.h

```diff
@@ -172,6 +172,11 @@ static inline void xtimer_spin(xtimer_ticks32_t ticks) {
     _xtimer_spin(ticks.ticks32);
 }
 
+static inline void xtimer_msleep(uint32_t milliseconds)
+{
+    _xtimer_tsleep64(_xtimer_ticks_from_usec64(milliseconds * US_PER_MS));
```

Comment:
Hm now I wonder: will this now have a worse overhead than calling `xtimer_usleep(ms * US_PER_MS)` because here 64 bit arithmetic will be involved?

Reply:
Rather than worrying about `xtimer`, let's push `ztimer` forward :-)